src/utils/loadScript.ts

```diff
@@ -34,7 +35,14 @@ const appendScript = (): HTMLScriptElement => {
 
 const loadScript = (loadMainCDN: boolean = true) => {
   const collectPromise = new Promise((resolve, reject) => {
-    scriptURL = loadMainCDN ? scriptURL : BACKUP_SCRIPT_DOMAIN;
+    const { version } = getConfig();
+
+    scriptURL = loadMainCDN
```

Comment:
I'd suggest avoiding ternary operators like that. It looks like you wrote less code with it, but it's harder to read (imho)

Reply:
Thank you! I've simplified logic :)